src/mon/OSDMonitor.cc

```diff
@@ -2248,22 +2248,23 @@ void OSDMonitor::share_map_with_random_osd()
 version_t OSDMonitor::get_trim_to() const
 {
   if (mon->get_quorum().empty()) {
-    dout(10) << __func__ << ": quorum not formed" << dendl;
+    dout(10) << __func__ << ": quorum not formed, trim_to = 0" << dendl;
```

Comment:
could you take the opportunity to drop the ":" to be more consistent with the rest of this file?